servers/physics_2d/godot_body_pair_2d.cpp

```diff
@@ -171,14 +171,16 @@ bool GodotBodyPair2D::_test_ccd(real_t p_step, GodotBody2D *p_A, int p_shape_A,
 
 	real_t min, max;
 	p_A->get_shape(p_shape_A)->project_rangev(mnormal, p_xform_A, min, max);
-	bool fast_object = mlen > (max - min) * 0.3; //going too fast in that direction
+	bool fast_object = mlen > (max - min) * 0.3; // Going too fast in that direction
```

Comment:
Nitpick but I'd remove the trailing comment and move this statement below the explanatory comment on the next line.

Reply:
As long as we are in the nitpicks, the point is missing :)

Reply:
Hehe yeah but my proposal solves that by removing the whole sentence :P